mobile-app/src/domain/alert/screens/alert.screen.tsx

```diff
@@ -29,33 +29,57 @@ const AlertScreen = () => {
 		location: {} as DeviceLocation,
 	});
 
+	const [previousAppState, setPreviousAppState] =
+		useState<AppStateStatus>('unknown');
+
 	const { fullName } = useUserInfoStore();
 	const { contacts } = useContactStore();
 
 	const [minutes, setMinutes] = useState(0);
 	const [hint, setHint] = useState<string>();
 
 	const {
-		permissionsGranted: locationPermissionsGranted,
+		permissionGranted: locationPermissionsGranted,
 		getHighPriorityLocation,
 		getLowPriorityLocation,
 	} = useLocation();
 
+	const updateLowPriorityLocationContext = () => {
+		getLowPriorityLocation().then(location => {
+			setContext((current) => {
```

Comment:
Ja bih ovaj state preimenovao da se zove location, koliko razumem to je to zapravo, čisto da se ne pomeša sa react context-om.

const [location, setLocation] = useState<DeviceLocation>();